### DISCUSSION TITLE
Display changelog on Dependency Dashboard

### DISCUSSION THREAD

**Discussion by gautamg795** [renovatebot/renovate#16659] [Request Help] [answered]
Is it possible to display the changelog for each approval-pending upgrade in the Dependency Dashboard (perhaps under a closed fold), rather than just on the PR?

Our intended flow is that the dashboard is used for approval of all upgrades, and once approved, the PR created by Renovate is automatically merged by Mergify according to our labels/rules. In this flow, it'd be much easier to see the changes in the update before approval. Currently we have to approve, see the PR that gets created, then remove our custom "queue" label (used by Mergify) if there's something in the changelog warranting further discussion.

**Comment by rarkins** [comment, marked as answer]
It's not practical due to issue size limitations. We are already hitting limits at times

**Reply by gautamg795** [reply]
Understood. Any suggestions on a half-measure solution for this workflow? Two things that come to mind (neither of which are current features) would be to hyperlink to the changelog if available from the dashboard issue, or to split the dashboard into one issue per dependency (this is okay with us because we don't use GitHub Issues, but we're probably in a minority)

**Reply by rarkins** [reply]
Longer term we plan to host all changelogs on a public site and can then link to them
